packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -30,8 +28,8 @@ let mockUseSimpleBreadcrumb = false;
 jest.mock('react-i18next', () => ({
   // this mock makes sure any components using the translate hook can use it without a warning being shown
   useTranslation: () => ({
-      t: (str: string) => str,
-    }),
+    t: (str: string) => str,
+  }),
 }));
 
 jest.mock('react-router-dom', () => {
```

Comment:
remove the mock of useParams and try to mock the window location object.
remove all mocks of all components